crates/gitbutler-tauri/src/cli.rs

```diff
@@ -0,0 +1,89 @@
+use std::path::Path;
+
+use crate::error::Error;
+use anyhow::Context;
+use tracing::instrument;
+
+#[tauri::command(async)]
+#[instrument(err(Debug))]
+pub fn install_cli() -> anyhow::Result<(), Error> {
+    #[cfg(windows)]
+    {
+        // TODO
+        return Err(anyhow::anyhow!(
+            "CLI installation is not supported on Windows. Please install manually."
+        ))
+        .map_err(Error::from);
+    }
+
+    let cli_path = get_cli_path()?;
+    let link_path = Path::new("/usr/local/bin/but");
+    let bin_dir = link_path.parent().context("Cant find bin dir")?;
+
+    if link_path.exists() {
```

Comment:
I would get `symlink_metadata(link_path)`. From there one can more easily handle the case that there is an actual binary in which case we probably want to say so and abort the attempt.

Right now we would also abort, but with an error that says something about failing to read a link.